eng/PrepareRelease.yml

```diff
@@ -8,7 +8,8 @@ stages:
   - job: PrepareReleaseJob
     displayName: Prepare release with Darc
     pool: 
-      vmImage: windows-latest
+      name: NetCore1ESPool-Internal
+      demands: ImageOverride -equals build.windows.10.amd64.vs2017

```

Comment:
Is 2017 preferred to 2022 or 2019?

Reply:
Doesn't matter. Just has to run a little bit of a build. I think the other parts of this repo were using 2017.

Reply:
All other build legs are using VS2019